setup.cfg

```diff
@@ -1,8 +1,61 @@
+[bdist_wheel]
+universal = 1
+
```

Comment:
The resulting dist shouldn't have the `py2` tag since it only declares support for Python 3.6+
```suggestion
```